pkg/status/status.go

```diff
@@ -38,12 +41,14 @@ var (
 )
 
 type Status struct {
-	Status Code             `json:"status"`
-	Detail *json.RawMessage `json:"detail,omitempty"`
+	Status  Code             `json:"status"`
+	Detail  *json.RawMessage `json:"detail,omitempty"`
+	Version string           `json:"version,omitempty"`
 }
 
 func New(healthy bool, detail interface{}) (Status, error) {
 	var s Status
+	s.Version = version.String()
```

Comment:
Any reason to assign this here? The assignment in the handler should be sufficient.